round4/fehra.md

```diff
@@ -0,0 +1,108 @@
+# Chrome Extension Idea: Budget Patrol
+
+## Authors
+
+Allie Fehr, Ivan Zou, Roberto Ligeralde, Arjun Shah
+
+## Problem Statement
+
+Our Chrome Extension makes it easier to stay within your budget when shopping online. Often, people create budgets to keep their spending smart but have difficulty tracking how their potential purchases fit into their budget or need to manually open up a spreadsheet and compare potential purchases to their budgeting goals. Our chrome extension removes the need to have another tab open and insteads sends pop up notifications to the user letting them know how the purchases of the product whose page they are currently on would fit into or exceed their budget.
+
+## Target Audience
+
+Our target audience is people who enjoy shopping and are on a budget. Specifically, it would be useful for current college students and new grads who may need to buy new furniture or clothing for wherever they are moving but want to stay within a set budget because they may have no job, a part-time job, or just started a full-time job and not saved up much.
+
+## Description
+
+The purpose of our Chrome Extension is to help users stay within their budget when shopping online. It will allow users to set budget categories (e.g. clothing, food, vacation) and monthly budget goals. While a user is browsing the web page of a product/service they might buy, the extension will find the price and provide a pop-up notification to the user informing them of how that purchase would affect their budget (e.g. "This item exceeds your __ budget by $__" or "You will have $__ remaining in your __ budget after this purchase").
+
+## Selling Points
+
+1. Pop-up notifications prevent having to spend time switching between product page and budget spreadsheet.
+2. Works across many big online retailers such as Amazon, Ebay, etc.
+3. Considers an item automatically into your budget when that item is purchased.
+4. Synchronizes with traditional budgeting tools like Google Sheets.
+5. Easy to adjust budgets, and provides recommended budget breakdowns.
```

Comment:
The idea in itself is very valuable and solves a big unmet need for many. However, I'm curious about the feasibility of breaking down a user's spending into granular categories or budgets. I can shop clothing, food, and accessories at e.g. Amazon or Ebay, how would you be able to know what detailed category a specific purchase would fit into in a scalable way?

Something to also think about is that instead of only telling a user that "you are exceeding your budget with this purchase", could you perhaps provide alternatives to users that are cheaper? 

The chrome extension "Shade" does the latter to suggest more "sustainable" fashion purchases based on what you currently are looking at: https://www.shadyclub.com/